### DISCUSSION TITLE
[Suggestions] Game, add an option for the hero to use artifact spells (and therefore also spell scrolls) without having a magic book

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#5445] [Ideas]
From an idea of @Eldorias-Piano-Covers.
Add an option for the hero to use artifact spells (and therefore also spell scrolls) without having a magic book:

"heroes: allow to cast spells from artifacts without a magic book"

**Comment by Branikolog** [comment]
I think the Book is a required condition on using spells. Since in H3 it worked the same. A nice restriction for a player to force him always buying books.

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> I think the Book is a required condition on using spells. Since in H3 it worked the same. A nice restriction for a player to force him always buying books.

That's why it's an option.
Lets the players have fun with the game. :)

**Reply by Branikolog** [reply]
Implementing such minor options, in my opinion, will turn the game into something close to wog. 
I'm not opposed to some optional features, but I expect them kind of more game changing and unique. That brings completely new experience to prayers, rather than changing a tiny rule, the majority of prayers even didn't think about. 
For instance, forbid all mage guilds and spell books, but allowing to use the spell that player has in particular scroll.

**Reply by LeHerosInconnu** [reply]
Have you ever played a scenario, in impossible mode, where you start with a Knight or Barbarian Hero in a Knight or Barbarian castle?
In this kind of scenario, the Mage Guild cannot be built until quite late in the game, with the construction of creature dwellings being a priority in view of the scarcity of construction resources.
In such a scenario, I can assure you that being able to use artifact spells without a magic book would be a completely different experience.
And it would look quite similar to your description:
"For instance, forbid all mage guilds and spell books, but allowing to use the spell that player has in particular scroll.".

**Reply by Branikolog** [reply]
Sure, I've played. ^_^
But I still find this proposal not essential for the majority of cases. Like allowing to hire not 8, but 9 heroes. This can hardly affect the gameplay experience, but list of such minor options can fill the entire screen, making players run away pop-eyed and screaming terrified. :)

**Reply by LeHerosInconnu** [reply]
I know you want a game with as few options as possible so it doesn't feel like the "wog" experience. :rofl: 
For me, I'm leaning more towards a game where the player can make adjustments to have a game that is fun to play for him.
Of course we can't put all kinds of options more or less desirable, but we should also take into account the demand of the players and if the option is interesting or practical, why shouldn't we make it available under the pretext that there are too many options.
I propose to leave this on hold and see if there are many players who would be interested in this option. :)

**Reply by zenseii** [reply]
I think enough changes to warrant a new game mode is a good compromise, in the future when we have the time to look at implementing such a thing 😂
